libvast/src/meta_index.cpp

```diff
@@ -197,4 +197,47 @@ caf::error inspect(caf::deserializer& source, meta_index& x) {
                 x.blacklisted_layouts_);
 }
 
+// Perform a deep equality comparison for meta indices. This is slow, don't
+// actually call this outside of unit tests.
```

Comment:
This sounds a bit like a footgun (i.e. imagine someone making an `unordered_set<meta_index>` in the future while not being aware of this comment), would it make sense to move this operator to `libvast/test/meta_index.cpp`?

Reply:
Can do.